00_Intro/bedrock_boto3_setup.ipynb

```diff
@@ -37,14 +39,14 @@
    },
    "outputs": [],
    "source": [
-    "# Make sure you ran `download-dependencies.sh` from the root of the repository first!\n",
     "%pip install --no-build-isolation --force-reinstall \\\n",
-    "    ../dependencies/awscli-*-py3-none-any.whl \\\n",
-    "    ../dependencies/boto3-*-py3-none-any.whl \\\n",
-    "    ../dependencies/botocore-*-py3-none-any.whl"
+    "    \"boto3>1.28.56\" \\\n",
+    "    \"awscli>=1.29.56\" \\\n",
```

Comment:
The latest version pre-GA is vx.x.56.

The GA release will likely be vx.x.**57**

Reply:
Fixed this. Updated to Vx.x.57